source/features/add-jump-to-bottom-link.js

```diff
@@ -13,3 +14,8 @@ export default function () {
 		<a href="#partial-timeline-marker" id="refined-github-jump-to-bottom-link">Jump to bottom</a>
 	);
 }
+
+export default function () {
```

Comment:
Can you use arrow function?

Reply:
Why?

Reply:
Why not. It's shorter and doesn't introduce a `this` binding.

Reply:
1. `this` is not an issue in this case, it's `undefined` either way.
1. I prefer `function` for exports because it's a declaration and not an expression (since exports are static as well)
	* so I use arrow functions only if they're _generated_ during the execution (e.g. if they have to **follow** other code or `const` assignments)
1. `const name = () => {multiline; return}` is actually longer than `function name() {multiline; return}`
	* so if there are only `function`s at the top level (like in this case) I just use declarations for all
1. ### `;`

Reply:
Arrow functions just feel lighter weight to me. Right now, they're not faster, but they will be in the future, although probably negligible for our uses. I also prefer arrow functions as they don't have weird hoisting like function statements. But I don't feel strongly about it.